src/core/definitions.jl

```diff
@@ -21,9 +21,17 @@ const JuMPParamArray = JuMP.Containers.DenseAxisArray{PJ.ParameterRef}
 const DenseAxisArrayContainer = Dict{Symbol, JuMP.Containers.DenseAxisArray}
 
 @enum BUILD_STATUS begin
-    BUILT = 1
+    BUILT = 0
     IN_PROGRESS = -1
-    EMPTY = 0
+    FAILED_BUILD = 1
+    EMPTY = 2
+end
+
+@enum RUN_STATUS begin
+    READY = -1
+    SUCCESSFUL_RUN = 0
+    RUNNING = 1
+    FAILED_RUN = 2
```

Comment:
I have a minor preference for sticking to the same logic for BUILD_STATUS and RUN_STATUS.

```suggestion
@enum RUN_STATUS begin
    RUNNING = -1
    READY = 0
    SUCCESSFUL_RUN = 1
    FAILED_RUN = 2
```

The idea being negative numbers for current state while running (`IN_PROGRESS`/`RUNNING`), zero for the success state, and positive numbers for the state after a process is finished (`FAILED_BUILD`/`FAILED_RUN`) 

But if you have a reason for doing it the current way, I'm happy to be overruled on this.

Reply:
addressed in 22c4fcf